terraform/aks/workspace_variables/qa_aks.tfvars.json

```diff
@@ -22,5 +22,6 @@
     "qa.find-postgraduate-teacher-training.service.gov.uk",
     "qa.api.publish-teacher-training-courses.service.gov.uk"
   ],
-  "enable_monitoring": true
+  "enable_monitoring": true,
+  "statuscake_contact_groups": [151103]
```

Comment:
Is this the Infra group or the Find & Apply group?

Reply:
This is the Find & Apply group. I've added the infra group to prod only. I don't want to get spammed!